requirements.txt

```diff
@@ -1,4 +1,4 @@
 flask
-jsonschema[format]
+jsonschema[format]==4.17.3
```

Comment:
Pinned down jsonschema version to 4.17.3 as tests fail otherwise. 4.18.0 actually passes tests but fails linter so needed to use 4.17.3